worldedit-core/src/main/java/com/sk89q/worldedit/extent/clipboard/io/share/ClipboardShareDestinations.java

```diff
@@ -0,0 +1,79 @@
+/*
+ * WorldEdit, a Minecraft world manipulation toolkit
+ * Copyright (C) sk89q <http://www.sk89q.com>
+ * Copyright (C) WorldEdit team and contributors
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
+package com.sk89q.worldedit.extent.clipboard.io.share;
+
+import com.sk89q.worldedit.WorldEdit;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Locale;
+import java.util.Map;
+import javax.annotation.Nullable;
+
+import static com.google.common.base.Preconditions.checkNotNull;
+
+public class ClipboardShareDestinations {
+
+    private static final Map<String, ClipboardShareDestination> aliasMap = new HashMap<>();
+    private static final List<ClipboardShareDestination> registeredDestinations = new ArrayList<>();
+
+    public static void registerClipboardShareDestination(ClipboardShareDestination destination) {
+        checkNotNull(destination);
+
+        for (String key : destination.getAliases()) {
+            String lowKey = key.toLowerCase(Locale.ENGLISH);
```

Comment:
weren't we using ROOT throughout? is there a reason to use ENGLISH here? (same applies to the clipboardformats class this was copied from - not sure why it uses english either. also one usage in GeneralCommands for some reason. throughout the repo there's 38 ROOTS and 2 ENGLISH (+1 english for the default i18n locale which makes sense))